ons_alpha/bulletins/models.py

```diff
@@ -164,3 +175,16 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+    @path("previous/v<int:version>/")
+    def previous_version(self, request, version):
+        try:
+            page_revision = Revision.objects.get(pk=version)
+        except Revision.DoesNotExist:
+            raise Http404 from None
```

Comment:
Suggestion: There's a `django.shortcuts.get_object_or_404` helper function which might be useful here.